src/gui/addnewtorrentdialog.cpp

```diff
@@ -99,6 +104,27 @@ namespace
         Path current = path;
         qint64 freeSpace = Utils::Fs::freeDiskSpaceOnPath(current);
 
+        // UNC path support on windows
+        #ifdef Q_OS_WIN
+        if (freeSpace < 0 && PathIsUNC(current.data().toStdWString().c_str()))
+        {
+            ULARGE_INTEGER FreeBytesAvailable = { 0 };
+            ULARGE_INTEGER TotalNumberOfBytes = { 0 };
+            ULARGE_INTEGER TotalNumberOfFreeBytes = { 0 };
```

Comment:
```suggestion
            ULARGE_INTEGER freeBytesAvailable = {0};
            ULARGE_INTEGER totalNumberOfBytes = {0};
            ULARGE_INTEGER totalNumberOfFreeBytes = {0};
```

Reply:
IIRC, both `totalNumberOfBytes` and `totalNumberOfFreeBytes` are redundant (unused) and can be omitted.